.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -1,59 +1,135 @@
+        1. Sign in on the Jellyfin web client, with an admin account, using a browser of your choice.
+        2. Navigate to the dashboard.
+        3. Select "general".
+        4. Change the login disclaimer to something like "I am a cool disclaimer!"
+        5. Save the settings.
+        6. Sign out.
+        7. Make sure you are on the sign in screen. Otherwise, navigate to the sign in screen manually.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: What is the current _bug_ behavior?
+      description: Write down the incorrect behavior that currently happens after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen has the server name appended to the text. The text shown is: "I am a cool disclaimer!jellyfinserver".
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What is the expected _correct_ behavior?
+      description: Write down the correct expected behavior that is supposed to happen after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen should only show the configured text. The text that should be shown is: "I am a cool disclaimer!".
     validations:
       required: true
   - type: dropdown
     id: version
     attributes:
-      label: Jellyfin Version
-      description: What version of Jellyfin are you running?
+      label: Jellyfin Server version
+      description: What version of Jellyfin are you using?
       options:
-        - 10.9.0
-        - 10.8.13
-        - 10.8.12 or older (please specify)
-        - Weekly unstable (please specify)
-        - Master branch
+        - 10.9.6
+        - Master
+        - Unstable
+        - Older*
+      default: 0
     validations:
       required: true
   - type: input
-    id: version-other
+    id: version-master
+    attributes:
+      label: "Specify commit id"
+      description: Fill in this field in case the option 'master' is selected. Provide the commit id it was built on.
+      placeholder: |
+        610e56baafc3011e1bfa043bdabb567bda0c2ab0
+  - type: input
+    id: version-unstable
+    attributes:
+      label: "Specify unstable release number"
+      description: Fill in this field in case the option 'unstable' is selected. Provide the unstable release number.
+      placeholder: |
+        2024050906
+  - type: input
+    id: version-older
+    attributes:
+      label: "Specify version number"
+      description: Fill in this field in case the option 'older' is selected. Provide the version number.
+      placeholder: |
+        x.y.z
+  - type: input
+    id: build-version
     attributes:
-      label: "if other:"
-      placeholder: Other
+      label: "Specify the build version"
+      description: Please provide the build version that is shown in the dashboard.
+      placeholder: |
+        x.y.z
```

Comment:
This will be a date time for unstable builds.

Reply:
I will just remove the placeholder in that case.